.travis.yml

```diff
@@ -1,13 +1,18 @@
 language: php
 
-php:
-  - 5.4
-  - 5.5
-  - 5.6
+sudo: false
+
+matrix:
+  include:
+    - php: 5.4
+      env:
+        - DEPENDENCIES='low'
+    - php: 5.6
```

Comment:
If it runs on PHP 5.6 and 5.4 then why wouldn't it on 5.5? As for 7.0, it's out of this PR's scope - this is a simple update.